Tests/test_image_resize.py

```diff
@@ -57,13 +57,13 @@ def test_convolution_modes(self):
             Image.Resampling.LANCZOS,
         ),
     )
-    def test_reduce_filters(self, resample):
-        r = self.resize(hopper("RGB"), (15, 12), resample)
```

Comment:
Why is 'resampling_filter' clearer? 'resample' is the name of the variable when the core resize operation is called in Image.py
https://github.com/python-pillow/Pillow/blob/86634b835257a7d3438eaf783d70eca12e20b13e/src/PIL/Image.py#L2192